modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -481,20 +481,16 @@
           end
         end
 
-        context 'when the city is invalid' do
+        context 'when the city is valid' do
```

Comment:
Should we also make a test in which the city is invalid? It seems like we are losing coverage.